src/main/java/in/ashwanthkumar/gocd/artifacts/Janitor.java

```diff
@@ -92,11 +93,19 @@ public void run(String pathToConfiguration, Boolean dryRun) throws IOException {
 
         long bytesProcessed = performAction(whiteList, config.getArtifactStorage(), dryRun);
 
-        LOG.info("Total bytes (deleted / moved) processed so far - " + FileUtils.byteCountToDisplaySize(bytesProcessed));
+        LOG.info("Total bytes (deleted / moved) - " + FileUtils.byteCountToDisplaySize(bytesProcessed));
         LOG.info("Shutting down Janitor");
     }
 
-    /* default */ List<PipelineConfig> pipelinesNotInConfiguration(GoCD client, final JanitorConfiguration config) throws IOException {
+    private List<PipelineConfig> getPipelines() throws IOException {
```

Comment:
I generally prefer to have all private methods towards the end of the class (in the file) - that way for someone when browsing the code - he's mostly looking into the public methods first - and private methods are generally super small and names are also self explanatory.